ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -53,7 +53,12 @@ open class NavigationBarTopSearchBarAttributes: NavigationBarTopAccessoryViewAtt
 /// Contains the MSNavigationTitleView class and handles passing animatable progress through
 /// Custom UI can be hidden if desired
 @objc(MSFNavigationBar)
-open class NavigationBar: UINavigationBar {
+open class NavigationBar: UINavigationBar, TwoLineTitleViewDelegate {
+    public func twoLineTitleViewDidTapOnTitle(_ twoLineTitleView: TwoLineTitleView) {
+        // TODO: This won't be needed once we move it to LargeTitleView
+        NotificationCenter.default.post(name: .accessoryExpansionRequested, object: self)
```

Comment:
Do we need to fix this before check-in or is it a post check in fix?

Reply:
Turns out we still do need it if we want to put a `TwoLineTitleView` in a vanilla navigation bar. Removing comment.